### DISCUSSION TITLE
Importing Road Runner Map in Carla " udatasmith "

### DISCUSSION THREAD

**Discussion by pratikvyas101** [carla-simulator/carla#6480] [General]
I have followed all steps in the below link from MathWorks
https://in.mathworks.com/help/roadrunner/ug/export-to-carla.html
But on step number 4 on selecting RRCarlaDataPrep we get a null error in terminal of Unreal Engine
LogAssetEditorSubsystem: Error: Opening Asset editor failed because of null asset

Please suggest a way to import a road runner map of udatasmith format in Carla

**Comment by lemon-lone** [comment]
I have the same problem as you. If it's solved, please let me know. Thank you very much
